lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -278,6 +291,54 @@ def test_not_found(self, mock_request):
         )
 
 
+@ddt.ddt
+@patch('requests.request')
+@override_settings(MODULESTORE=TEST_DATA_MONGO_MODULESTORE)
+class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
+    """
+    Ensures the number of modulestore queries is deterministic based on the
+    number of responses retrieved for a given discussion thread.
+    """
+
+    @ddt.data(
+        # old mongo: number of responses plus 16.  TODO: O(n)!
+        (ModuleStoreEnum.Type.mongo, 1, 17),
+        (ModuleStoreEnum.Type.mongo, 50, 66),
+        # split mongo: 3 queries, regardless of thread response size.
+        (ModuleStoreEnum.Type.split, 1, 3),
+        (ModuleStoreEnum.Type.split, 50, 3),
+    )
+    def test_number_of_split_queries(self, store_responses_and_calls, mock_request):
```

Comment:
You could do `@ddt.unpack` and then change the arg list to `self, default_store, num_thread_responses, num_mongo_calls, mock_request` I think. Not important but maybe easier to read.